src/openrct2/localisation/Localisation.cpp

```diff
@@ -1337,7 +1338,7 @@ sint32 win1252_to_utf8(utf8string dst, const char *src, size_t srcLength, size_t
     utf16 stackBuffer[256];
     utf16 *heapBuffer = NULL;
     utf16 *intermediateBuffer = stackBuffer;
-    size_t bufferCount = countof(stackBuffer);
+    size_t bufferCount = Util::CountOf(stackBuffer);
     if (maxBufferLength > bufferCount) {
         if (srcLength > bufferCount) {
             bufferCount = srcLength + 4;
```

Comment:
`malloc` below will trigger error as well. You could change it to `new` and `delete` few lines further and remove the `assert`